tests/src/test/java/alluxio/client/cli/job/DistributedCommandsStatsTest.java

```diff
@@ -56,12 +61,24 @@
                 .setExistsMs(SLEEP_MS)
                 .setMkdirsMs(SLEEP_MS)));
 
+  @Rule
+  public TemporaryFolder mTempFolder = new TemporaryFolder();
+
+  private String mLocalUfsPath;
+  private FileSystem mFileSystem;
+
+  @Before
+  public void before() throws Exception {
+    mLocalUfsPath = mTempFolder.getRoot().getAbsolutePath();
+    sFileSystem.mount(new AlluxioURI("/mnt/"), new AlluxioURI("sleep://" + mLocalUfsPath));
+  }
+
   @Test
   public void testCompleteStats() throws Exception {
     final int length = 10;
-    FileSystemTestUtils.createByteFile(sFileSystem, "/test", WritePType.THROUGH, length);
+    FileSystemTestUtils.createByteFile(sFileSystem, "/mnt/test", WritePType.THROUGH, length);
 
-    long jobId = sJobMaster.run(new LoadConfig("/test", 1, Collections.EMPTY_SET,
+    long jobId = sJobMaster.run(new LoadConfig("/mnt/test", 1, Collections.EMPTY_SET,
```

Comment:
why do we need sleep ufs for this test?

Reply:
yeah we don't need sleeping ufs for it, maybe we can remove this test? Previously adding this test to test the function call correctness if remember correctly.